modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/util/Locator.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.taglib.clay.servlet.taglib.util;
+
+import java.util.HashMap;
+
+/**
+ * @author Julien Castelain
+ */
+public class Locator extends HashMap<String, String> {
```

Comment:
`Locator` is maybe not the best name for this when you find it out of context. Maybe `MultiSelectLocator`?

Reply:
Yes, we've changed that to `MultiSelectLocator`, I think @kresimir-coko will add a link to the commit to confirm

Reply:
Hey @carloslancha I've fixed this in [fb66632](https://github.com/liferay-frontend/liferay-portal/pull/528/commits/fb66632ed71690f56b46829b6099155275326d92).